pkg/hintrunner/zero/zerohint_signatures_test.go

```diff
@@ -26,5 +26,17 @@ func TestSignatures(t *testing.T) {
 				},
 			},
 		},
+		"ImportSecp256R1P": {
+			{
+				operanders: []*hintOperander{},
+				ctxInit: func(ctx *hinter.HintRunnerContext) {
+					ctx.ScopeManager.EnterScope(map[string]any{})
```

Comment:
I would rather use here `InitializeScope` function, as I think up until this function is initialized the ScopeManager is still nil, right?